### PR TITLE
[EPMRPP-38819] Update reporting

### PR DESCRIPTION
**Pull request by ihar-kahadouski**

### CODE REVIEW

File: src/main/java/com/epam/ta/reportportal/core/launch/TerminateLaunchHandler.java [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+import static com.epam.ta.reportportal.commons.validation.BusinessRule.expect;
+import static com.epam.ta.reportportal.commons.validation.Suppliers.formattedSupplier;
+import static com.epam.ta.reportportal.entity.enums.StatusEnum.*;
+import static com.epam.ta.reportportal.entity.project.ProjectRole.PROJECT_MANAGER;
+import static com.epam.ta.reportportal.ws.model.ErrorType.*;
+
+/**
+ * @author <a href="mailto:ihar_kahadouski@epam.com">Ihar Kahadouski</a>
+ */
+public interface TerminateLaunchHandler {
```
Comment (pbortnik) [inline src/main/java/com/epam/ta/reportportal/core/launch/TerminateLaunchHandler.java]: it contains only validation methods, so it is probably better to rename it

Comment (ihar-kahadouski) [reply src/main/java/com/epam/ta/reportportal/core/launch/TerminateLaunchHandler.java]: Provide please your alternative

Comment (pbortnik) [reply src/main/java/com/epam/ta/reportportal/core/launch/TerminateLaunchHandler.java]: LaunchValidator
and it is probably better to make it as an util class ? 
it's not really interface as it is

Comment (ihar-kahadouski) [reply src/main/java/com/epam/ta/reportportal/core/launch/TerminateLaunchHandler.java]: fixed

**Pull request merged**